packages/website/src/components/Uploader/Uploader.js

```diff
@@ -47,15 +47,37 @@ const LogInLink = () => {
   );
 };
 
+const SubscribeLink = () => {
+  const createAccountsUrl = useAccountsUrl();
+
+  return (
+    <Link
+      href={createAccountsUrl("payments")}
+      className="uppercase underline-primary hover:text-primary transition-colors duration-200"
+    >
+      available plans
+    </Link>
+  );
+};
+
 const Uploader = () => {
   const [mode, setMode] = React.useState("file");
   const [uploads, setUploads] = React.useState([]);
 
   const { data: accounts } = useAccounts();
-  const showAccountFeatures =
-    accounts && accounts.enabled !== false && !accounts?.auth_required && !accounts?.authenticated;
-  const disabledComponent =
-    accounts && accounts.enabled !== false && accounts?.auth_required && !accounts?.authenticated;
+
+  // variables extracted from useAccounts response
+  const isAccountsEnabled = accounts?.enabled === true;
```

Comment:
`nit` These return boolean values, right?

`=== true` feels over the top, but maybe I'm missing something.

Reply:
I usually like to enforce boolean when dealing with variables that I use in jsx comparisons to avoid any mistakes that could potentially print a variable value. Although in this case I think it's safe to drop those - https://github.com/SkynetLabs/skynet-webportal/pull/1852/commits/18822edae8bb867dafbee4cc97258ea7bde74ce8